docs/src/main/asciidoc/cache-infinispan-reference.adoc

```diff
@@ -209,4 +209,4 @@ the cache.
 quarkus.cache.infinispan.my-cache.max-idle=100s
 ----
 
```

Comment:
what is all these docs include changes about? they weren't there earlier and doesn't seem related. mistake or expected?

Reply:
What do you mean, they weren't there earlier? I'm updating existing includes all over the place? Or can you point me to the place where I add some?

The reason why the naming is different is that the annotation processor is a lot smarter now and the naming used to be derived from the config root name with a lot of weird corner cases.

The naming is now a lot more consistent.

Reply:
it just that the PR went from 520ish to 620ish files changed so wanted to grok why.

if its because the generated files are now different that explains it.

Reply:
Well, some of them are the same. All the ones that were generated weirdly and whose name was beyond logic are now following a sane convention.